cspell.json

```diff
@@ -0,0 +1,90 @@
+{
+  "allowCompoundWords": true,
+  "caseSensitive": false,
+  "language": "en,ru,uk",
+  "minWordLength": 3,
+  "dictionaryDefinitions": [
+    {
+      "addWords": true,
+      "name": "custom-dict",
+      "path": "cspell/custom-dict.txt"
+    }
+  ],
+  "dictionaries": [
+    "aws",
+    "bash",
+    "companies",
+    "cpp",
+    "css",
+    "custom-dict",
+    "data-science",
+    "docker",
+    "dotnet",
+    "en_us",
+    "en-common-misspelling",
+    "fonts",
+    "fullstack",
+    "golang",
+    "html",
+    "k8s",
+    "makefile",
+    "misc",
+    "npm",
+    "people-names",
+    "python",
+    "ru_RU",
+    "rust",
+    "software-terms",
+    "uk_UA",
+    "win32"
+  ],
+  "enabled": true,
+  "files": ["**/*.{example,md}"],
+  "ignorePaths": [
+    ".dockerignore",
+    ".github",
+    ".gitignore",
+    "*.excalidraw",
+    "*.yml",
+    "app/**",
+    "CODE_OF_CONDUCT_RU.md",
+    "cspell.json",
+    "cspell/**",
+    "dashboards/**",
+    "deployment/**",
+    "docs/Gemfile",
+    "docs/googlec3812dcf278679ec.html",
+    "go.mod",
+    "go.sum",
+    "lib/**",
+    "Makefile",
+    "package/**",
+    "ports/**",
+    "snap/**",
+    "vendor/**"
+  ],
```

Comment:
I think we should simply limit spell check to `docs` folder. Then we can remove all these exclusions.

Reply:
That's a good point. I've fixed a misconfiguration in my code that allows to use this approach.
Thanks!